src/PageNavigation/NextPageLink.tsx

```diff
@@ -0,0 +1,19 @@
+import React from '@portal/react';
```

Comment:
I would merge it after other PRs and remove this reexport.

Reply:
I will rebase it one I merge this: https://github.com/Redocly/portal-default-theme/pull/9